src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -2351,7 +2344,7 @@ namespace OpenMS
       ExperimentType exp;
       try
       {
-        if (!fh.loadExperiment(*it, exp))
+        if (!fh.loadExperiment(*it, exp, {}, ProgressLogger::NONE, true, true))
```

Comment:
here as well... you explicitly set true,true
and I recall some places were you set it to false,false

... just worth a second glance